docs/serving/installing-istio.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "Performing a Custom Istio Installation"
+weight: 15
+type: "docs"
+---
+
+Use this guide to perform a custom installation of Istio for use with Knative.
+
+## Download Istio
+
+Run below command to download Istio.
+```shell
+# Download and unpack Istio
+ISTIO_VERSION=1.1.3
+DOWNLOAD_URL=https://github.com/istio/istio/releases/download/${ISTIO_VERSION}/istio-${ISTIO_VERSION}-linux.tar.gz
+
+wget $DOWNLOAD_URL
+tar xzf istio-${ISTIO_VERSION}-linux.tar.gz
+
+cd istio-${ISTIO_VERSION}
+```
+
+## Install Istio CRDs 
+Default Istio Installation
+Run below command to install Istio CRDs first.
+```shell
+for i in install/kubernetes/helm/istio-init/files/crd*yaml; do kubectl apply -f $i; done
+```
+wait a few seconds for the CRDs to be committed in the Kubernetes API-server
+
+## Custom Installation
+
+### Istio with Sidecar Injector
```

Comment:
To @sebgoa 's point, can we offer a recommend install (without using helm) that will be appropriate for most use cases? Having to make a choice about what Istio to install before even attempting to install Knative could really discourage people; we should provide a clear, easy path to get started.

Reply:
Hm, even in that IKS doc, the "manual instructions" point to installing Istio using our provided files, which are going away. From that place in the guide, I'd like to point users who want to perform a manual install to this page to install Istio.

Reply:
The Istio file (https://raw.githubusercontent.com/knative/serving/v0.5.2/third_party/istio-1.0.7/istio.yaml) won't go away as it is a part of v0.5.2 which has been released. So the instruction in IKS will still be valid.
But we won't have any Istio yaml in our 0.6.0 release.

Reply:
Yes, and since we won't have Istio .yaml in the 0.6 release, we need to update all the docs for 0.6 to point to these manual instructions for install, from the various install guides where no managed option exists. We won't point back to the 0.5.2 istio .yamls from the 0.6 docs. That's something I can take care of after we merge this page in, though.